src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory_Methods.vb

```diff
@@ -71,19 +71,33 @@ Namespace Microsoft.CodeAnalysis.Semantics
             Return arguments.ToImmutableAndFree()
         End Function
 
+        Private Function CreateConversion(expression As BoundExpression) As Conversion
+            If expression.Kind = BoundKind.Conversion Then
```

Comment:
@AlekseyTs It just occurs to me, do we need a null check here for `expression`? A quick look at the binder code that creates `BoundByRefArgumentWithCopyBack` seems to suggest this is not necessary

Reply:
> A quick look at the binder code that creates BoundByRefArgumentWithCopyBack seems to suggest this is not necessary

I agree.